apis/kueue/webhooks/workload_webhook.go

```diff
@@ -161,6 +166,30 @@ func validatePodSetName(name string, fldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
+func validateAdmission(obj *kueue.Workload, path *field.Path) field.ErrorList {
+	admission := obj.Spec.Admission
+	var allErrs field.ErrorList
+	allErrs = append(allErrs, validateNameReference(string(admission.ClusterQueue), path.Child("clusterQueue"))...)
+
+	flavorPath := path.Child("podSetFlavors")
+	for i, flavor := range admission.PodSetFlavors {
+		allErrs = append(allErrs, validateNameReference(flavor.Name, flavorPath.Index(i).Child("name"))...)
```

Comment:
```suggestion
		allErrs = append(allErrs, validatePodSetName(flavor.Name, flavorPath.Index(i).Child("name"))...)
```

Although, maybe we don't need this validation, given than the value should be present in the podSets, which are already validated.

Reply:
Yes, updated.